src/components/Articles/CompanyProvisions.vue

```diff
@@ -229,15 +223,22 @@ export default class CompanyProvisions extends Mixins(CommonMixin) {
   margin-left: 0;
 }
 
-::v-deep .v-input--selection-controls {
-  margin-top: 1.875rem;
-  padding: 0;
-}
-
-.checkbox-label {
+::v-deep .v-input--checkbox .theme--light.v-label {
```

Comment:
Do you still need the styling on line 220 above?

Reply:
Yes, I do need them.